lib/rubocop/cop/style/hash_syntax.rb

```diff
@@ -92,17 +92,20 @@ module Style
       #
       # @example EnforcedShorthandSyntax: consistent
       #
-      #   # bad
-      #   {foo: , bar: bar}
-      #
       #   # good
       #   {foo:, bar:}
       #
-      #   # bad
-      #   {foo: , bar: baz}
+      #   # good - can't omit `qux`
+      #   {foo: foo, bar: qux}
```

Comment:
After `foo` and `bar`, `baz` is common. Not `qux`.
https://www.rfc-editor.org/rfc/rfc3092.txt

```suggestion
      #   # good - can't omit `baz`
      #   {foo: foo, bar: baz}
```